pymathics/natlang/main.py

```diff
@@ -1223,23 +1242,24 @@ def eval(self, word, evaluation: Evaluation, options: dict):
             return
 
         senses = self._senses(py_word, wordnet, language_code)
-        return ListExpression(*[[String(s) for s in desc] for syn, desc in senses])
+        if senses is not None:
+            return ListExpression(*[[String(s) for s in desc] for syn, desc in senses])
 
     def eval_property(self, word, property, evaluation: Evaluation, options: dict):
-        "WordData[word_, property_String, OptionsPattern[%(name)s]]"
-        if word.get_head_name() == "System`StringExpression":
+        "WordData[word_, property_String, OptionsPattern[WordData]]"
+        if word.get_head is SymbolStringExpression:
             if property.get_string_value() == "Lookup":
                 return Expression(SymbolDictionaryLookup, word)
-        elif isinstance(word, String) or word.get_head_name() == "System`List":
+        elif isinstance(word, String) or word.get_head() is SymbolList:
```

Comment:
It seems like there is an inconsistency in `get_head sometimes it is an attribute and sometimes it is a method.
BTW, in the attribute case I think it should be `head` instead of `get_head`

Reply:
`head` is an attribute for `Expression`, while `get_head` is a method in all the daughter classes of `BaseElement`

Reply:
On Atoms we don't have a "head" property. We should. Maybe  after 6.0.0 we will correct that.